tests/cpp/utils.cpp

```diff
@@ -17,6 +17,25 @@
 
 namespace nvfuser {
 
+CGResultsPackage scheduleAndRun(
+    Fusion* fusion,
+    SchedulerType scheduler_type,
+    const at::ArrayRef<c10::IValue>& runtime_inputs,
+    bool validate_scheduler) {
+  auto heuristic_params = SchedulerEntry::scheduleWith(
+      fusion, scheduler_type, runtime_inputs, validate_scheduler);
+  auto fusion_executor = std::make_unique<FusionExecutor>();
+  fusion_executor->compileFusion(
+      fusion, runtime_inputs, heuristic_params->lparams);
```

Comment:
Don't we need to do this?

```suggestion
  fusion_executor->compileFusion(
      fusion, runtime_inputs, heuristic_params->lparams, heuristic_params->cparams);
```

Reply:
As we don't typically do anything to cparams in the schedulers, unlikely, though it doesn't hurt. I'm going to merge this to get through the second one. I'd like to get rid of the branch I'm on so the PR commit history doesn't look so long. If you'd like me to remove it I can do a small follow up PR.

Reply:
The normalization schedulers do use `cparams`. For example, https://github.com/NVIDIA/Fuser/blob/main/csrc/scheduler/normalization_inner.cpp#L413.

It's for performance optimization and shouldn't affect correctness, so it doesn't matter in these unit tests. Still, I don't think we should just ignore it.